### PR TITLE
Update community menu

### LINKED ISSUE #3294: beta updates: update community menu build

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update community menu

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Open the community menu
- See that it matches the [copy doc](https://docs.google.com/document/d/1TgsBJdGgsP4KrGiz5kU5QYkdGSlKqhAVnZGnCJgshYY/edit)

## Issue / Card

Fixes #3294

### CODE REVIEW

File: templates/templates/_navigation-community-h.html [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
-        <p class="p-p--small">Get the latest Ubuntu news and resources on our <a href="http://insights.ubuntu.com/" title="Visit Ubuntu Insights - external site">insights blog</a>.</p>
+        <h4 class="is-dense">Participate</h4>
+        <p class="p-p-small">The Ubuntu Community Hub is the meeting point for contributors and the Ubuntu community.</p>
+        <p><a class="p-button--positive" href="https://community.ubuntu.com/">Ubuntu community hub</a></p>
+        <ul class="p-text-list--small is-bordered">
+          <li class="p-list__item"><a href="https://askubuntu.com/">Answering questions on Ask Ubuntu</a></li>
+          <li class="p-list__item"><a href="https://ubuntuforums.org/">Join the forums</a></li>
+          <li class="p-list__item"><a href="https://community.ubuntu.com/t/translations/32">Translate apps into your first language</a></li>
+          <li class="p-list__item"><a href="https://wiki.ubuntu.com/DocumentationTeam/SystemDocumentation">Writing documentation</a></li>
+          <li class="p-list__item"><a href="https://www.ubuntu.com/download/desktop/thank-you">Ubuntu is free of charge, buy you can donate</a></li>
```
Comment (anthonydillon) [inline templates/templates/_navigation-community-h.html]: Ubuntu is free of charge, **but** you can donate

Comment (anthonydillon) [reply templates/templates/_navigation-community-h.html]: Please make this link relative

File: templates/templates/_navigation-community-h.html [hunk 2]
```diff
@@ -1,34 +1,57 @@
...
-        <p class="p-p--small"><a class="p-link--strong" href="https://community.ubuntu.com/t/internet-relay-chat/24/1" title="Visit the Ubuntu Community - external site">Internet Relay Chat (IRC)&nbsp;&rsaquo;</a><br>
-        Help people navigate the Ubuntu world by joining an Internet Relay Chat such as #ubuntu</p>
-        <p class="p-p--small"><a class="p-link--strong" href="https://community.ubuntu.com/t/community-support/709" title="Visit the Ubuntu Community - external site">More community support&nbsp;&rsaquo;</a><br>
-        Find support from a variety of sources</p>
+        <h4 class="is-dense">Ethos</h4>
+        <p class="p-p--small">The mission of Ubuntu is to bring the benefits of free software to the widest possible audience.</p>
+        <p><a class="p-button--positive" href="https://community.ubuntu.com/">Read about our open source mission</a></p>
+        <ul class="p-text-list--small is-bordered">
+          <li class="p-list__item"><a href="https://www.ubuntu.com/community/code-of-conduct">Ubuntu Code of Conduct</a></li>
+          <li class="p-list__item"><a href="https://www.ubuntu.com/community/diversity">The Ubuntu project welcomes everyone</a></li>
```
Comment (anthonydillon) [inline templates/templates/_navigation-community-h.html]: Relative please

File: templates/templates/_navigation-community-h.html [hunk 3]
```diff
@@ -1,34 +1,57 @@
...
-        <h4 class="is-dense">Connect</h4>
-        <p class="p-p--small"><a class="p-link--strong" href="https://community.ubuntu.com/t/internet-relay-chat/24/1" title="Visit the Ubuntu Community - external site">Internet Relay Chat (IRC)&nbsp;&rsaquo;</a><br>
-        Help people navigate the Ubuntu world by joining an Internet Relay Chat such as #ubuntu</p>
-        <p class="p-p--small"><a class="p-link--strong" href="https://community.ubuntu.com/t/community-support/709" title="Visit the Ubuntu Community - external site">More community support&nbsp;&rsaquo;</a><br>
-        Find support from a variety of sources</p>
+        <h4 class="is-dense">Ethos</h4>
+        <p class="p-p--small">The mission of Ubuntu is to bring the benefits of free software to the widest possible audience.</p>
+        <p><a class="p-button--positive" href="https://community.ubuntu.com/">Read about our open source mission</a></p>
+        <ul class="p-text-list--small is-bordered">
+          <li class="p-list__item"><a href="https://www.ubuntu.com/community/code-of-conduct">Ubuntu Code of Conduct</a></li>
```
Comment (anthonydillon) [inline templates/templates/_navigation-community-h.html]: Relative please

**Comment by anasereijo** [comment]
@anthonydillon 
- font size should be 14px
- can you make paddings and margins between elements (and also top and bottom) match what we have on Enterprise?

**Comment by steverydz** [comment]
@anthonydillon @anasereijo Fixed

**Comment by frankiegranato** [comment]
Small small text edits in the copydoc (sorry guys!!)

**Comment by steverydz** [comment]
@frankiegranato Copy amends done

**Comment by steverydz** [comment]
@frankiegranato Copy updates done

**Comment by frankiegranato** [comment]
Nice

**Comment by anasereijo** [comment]
@steverydz font size is still 16px and should be 14px. thanks

**Comment by steverydz** [comment]
@anasereijo Oh I thought you meant the buttons. Fixed now.

**Pull request merged**